jobs/web/spec

```diff
@@ -816,6 +836,13 @@ properties:
       `check_every` on the resource config.
     default: 1m
 
+  default_check_interval_with_webhook:
+    env: CONCOURSE_RESOURCE_WITH_WEBHOOK_CHECKING_INTERVAL
+    description: |
+      The interval, in Go duration format (1m = 1 minute), on which to check
+      for new versions of resources which have a webhook token configured.
+    default: 1m
```

Comment:
```suggestion
    example: 1m
```

we've been moving away from having defaults in the bosh release so that we let Concourse itself pick the defaults